### PR TITLE
Navbar router

### PR DESCRIPTION
**Pull request by tbrinkman3**
Added router functionality to navbar

### CODE REVIEW

File: client/src/components/Navbar.jsx [hunk 1]
```diff
@@ -12,24 +14,31 @@ const Navbar = (props) => {
             <BiHomeAlt/>
           </div>
         </IconContext.Provider>
-        <span className='navbar-button-text'>Portfolio</span>
+
+          <Link to="/login" className='navbar-button-text'>Portfolio</Link>
```
Comment (NylonEric) [inline client/src/components/Navbar.jsx:18]: Can you put the Link as a wrapper for the entire button area? currently you have to click the text to change the route. I put a suggestion revision above. The font of the "Portfolio" title looks different but the clicking action works better that way.

Example: https://stackoverflow.com/questions/42463263/wrapping-a-react-router-link-in-an-html-button

File: client/src/components/Navbar.jsx [hunk 2]
```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import {IconContext} from 'react-icons';
 import {BiHomeAlt, BiSearch, BiClipboard} from 'react-icons/bi';
 
+import {BrowserRouter, Link} from 'react-router-dom';
+
 
 const Navbar = (props) => {
   return (
```
Comment (NylonEric) [inline client/src/components/Navbar.jsx:9]:
```suggestion
  return (
     <div className='navbar-container'>
      <Link to="/login" className='navbar-button-text'>
      <div className='navbar-button'>
        <IconContext.Provider value={{className: 'navbar-button'}} >
          <div>
            <BiHomeAlt/>
          </div>
        </IconContext.Provider>
{/* <Link to="/login" className='navbar-button-text'>Portfolio</Link> */}
Portfolio
      </div>
      </Link>
```

**Review by NylonEric — CHANGES_REQUESTED** [review]
I would wrap the buttons in the link so clicking anywhere in the button area effects the route change. Looks good otherwise.

File: client/src/components/Navbar.jsx (hunk 1 above)
Comment (tbrinkman3) [reply client/src/components/Navbar.jsx:18]: Sure thing, I will take care of that this afternoon.  I thought I had done that, but I guess I just imagined that I did it.....

**Review by NylonEric — APPROVED** [review]
Works great!

**Pull request merged**